mods/tnt/init.lua

```diff
@@ -160,6 +162,43 @@ local function add_effects(pos, radius)
 		maxsize = 16,
 		texture = "tnt_smoke.png",
 	})
+
+	local item, texture = "", "tnt_blast.png" --fallback texture
```

Comment:
Minor nitpick: `item` can be local to the `if obj and ...` block below, since it's not used anywhere else.